src/cluster.ts

```diff
@@ -15,7 +15,55 @@
  */
 
 import {promisifyAll} from '@google-cloud/promisify';
-import * as is from 'is';
+import {CallOptions, Operation as GaxOperation} from 'google-gax';
+import {ServiceError} from 'grpc';
+
+import {google as btTypes} from '../proto/bigtable';
+import {Bigtable} from '.';
+import {Instance} from './instance';
+
+export interface GenericCallback<T> {
+  (err?: ServiceError|null, apiResponse?: T|null): void;
+}
+export interface GenericClusterCallback<T> {
```

Comment:
Not sure if this is necessary - I'm thinking that if these callbacks are only used once then a Generic isn't needed. Same in the cases where there are multiple callbacks with the same apiResponse. Let me know what you think!